opentelemetry-sdk/src/opentelemetry/sdk/metrics/__init__.py

```diff
@@ -48,47 +49,27 @@ def __init__(
         self,
         value_type: Type[metrics_api.ValueT],
         enabled: bool,
-        aggregator: Aggregator,
+        labels: Tuple[Tuple[str, str]],
+        metric: metrics_api.MetricT,
     ):
-        self.value_type = value_type
-        self.enabled = enabled
-        self.aggregator = aggregator
-        self._ref_count = 0
-        self._ref_count_lock = threading.Lock()
+        self._value_type = value_type
+        self._enabled = enabled
+        self._labels = labels
+        self._metric = metric
 
     def _validate_update(self, value: metrics_api.ValueT) -> bool:
-        if not self.enabled:
+        if not self._enabled:
             return False
-        if not isinstance(value, self.value_type):
+        if not isinstance(value, self._value_type):
             logger.warning(
-                "Invalid value passed for %s.", self.value_type.__name__
+                "Invalid value passed for %s.", self._value_type.__name__
             )
             return False
         return True
 
     def update(self, value: metrics_api.ValueT):
-        self.aggregator.update(value)
-
-    def release(self):
-        self.decrease_ref_count()
-
-    def decrease_ref_count(self):
-        with self._ref_count_lock:
-            self._ref_count -= 1
-
-    def increase_ref_count(self):
-        with self._ref_count_lock:
-            self._ref_count += 1
-
-    def ref_count(self):
-        with self._ref_count_lock:
-            return self._ref_count
-
-    def __repr__(self):
-        return '{}(data="{}")'.format(
-            type(self).__name__, self.aggregator.current
-        )
-
+        # The view manager handles all updates to aggregators
+        view_manager.update_view(self._metric, self._labels, value)
```

Comment:
qn: When user records a value for a instrument, do we still need to lookup the corresponding boundinstrument for the given keys? Or we first check if a View is registered for the instrument, and if none, simply drop it?

Reply:
Good question. I have not implemented any logic regarding the specified keys in the `View` nor any configuration (drops keys, keep all, etc.). However your question is valid in the design whether we are treating Views with different keys but the same aggregation and metric the same or not. See [my comment](https://github.com/open-telemetry/opentelemetry-python/pull/596#discussion_r411819266) here about this.

Reply:
Following discussion in the metrics SIG, we have decided to use a default aggregation if a view is not registered for a metric.